Loop/Managers/DeviceDataManager.swift

```diff
@@ -683,10 +683,10 @@ extension DeviceDataManager {
         pumpManager.enactBolus(units: units, automatic: automatic) { (error) in
             if let error = error {
                 self.log.error("%{public}@", String(describing: error))
-                
                 switch error {
                 case .uncertainDelivery:
                     // Do not generate notification on uncertain delivery error
```

Comment:
[nit] Is this comment still accurate? It looks like the `checkDeliveryUncertaintyState()` call cal post an alert.

Reply:
I reviewed this again, and I think the way it was in Tidepool is correct.